sysinfo bridge (#283)
awslabs/metrique: metrique-util/README.md

- `state`: Provides [`State<T>`], an atomically swappable shared value with snapshot-on-first-read semantics. Useful for shared runtime state (feature flags, config reloads, routing tables) that should appear on every metric record.
- `tokio-metrics-bridge`: Subscribes [tokio-metrics] runtime snapshots to a global entry sink. The reporter task is automatically aborted when the `AttachHandle` is dropped.

> - `sysinfo-bridge`: Subscribes [sysinfo] system and current-process snapshots to a global entry sink — CPU usage, memory and swap, load average, uptime, and current-process memory/CPU/disk/runtime/open-file counters. Disk space, network rx/tx, and thermal sensor metrics are opt-in via `SysinfoMetricsConfig::with_disks` / `with_networks` / `with_components`. The reporter task is automatically aborted when the `AttachHandle` is dropped.

Reviewer: ```suggestion
- `sysinfo-bridge`: Subscribes [sysinfo] system and current-process snapshots to a global entry sink, capturing metrics like CPU usage, disk space, and network rx/tx. The reporter task is automatically aborted when the `AttachHandle` is dropped.
```
